ee/controllers/phdeployment_controller.go

```diff
@@ -282,6 +282,23 @@ func (r *PhDeploymentReconciler) createDeployment(ctx context.Context, phDeploym
 	return nil
 }
 
+func compareEnvVar(a []corev1.EnvVar, b []corev1.EnvVar) bool {
```

Comment:
Should we deal with the case that same envs but different orders here?

Reply:
Will sort the []EnvVar at the beginning of Reconcile function